lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -15,7 +15,7 @@
 author: "Dario Zanzico (@dariko), Jason Witkowski (@jwitko)"
 short_description: docker swarm service
 description:
-  - Manage docker services. Allows live altering of already defined services.
+  - Manage docker services.
```

Comment:
How about putting `Swarm` in this sentence? :)

Reply:
(And that you need to talk with a manager node of a docker swarm.)

Reply:
Change it to:
> Manages docker services via a swarm manager node.